src/pages/students/[student_id]/iep.tsx

```diff
@@ -0,0 +1,15 @@
+interface StudentIEPProps {
+  first_name: string | undefined;
+  last_name: string | undefined;
```

Comment:
🐧 convention in TS for possible undefined/missing values is to do
```suggestion
  first_name?: string;
  last_name?: string;
```